msgvis/static/SparkQs/controllers.js

```diff
@@ -0,0 +1,144 @@
+(function () {
+    'use strict';
+
+    var module = angular.module('SparkQs.controllers', [
+
+    ]);
+
+    var requires = ['$scope'
+      ];
+    var DimensionController = function ($scope) {
+
+        $scope.$watch(function(){
+            console.log("digest called");
+        });
+
+        $scope.dimension_groups = [
+            {
+                "group_name": "Time",
+                "dimensions": [
+                    {
+                        "name": "Time",
+                        "type": "TimeDimension"
+                    },
+                    {
+                        "name": "Timezone",
```

Comment:
Actually lets use something like the [dimension API object](http://textvisdrg.readthedocs.org/en/latest/apps/api.html#msgvis.apps.api.serializers.DimensionSerializer) if that makes sense.